templates/base.twig

```diff
@@ -41,7 +41,7 @@
 	{% endif %}
 
 	{% block footer %}
-		{% include 'footer.twig' with { 'nav_type' : custom_styles.nav_type } %}
+		{% include 'footer.twig' with { 'nav_type' : custom_styles.nav_type ? 'minimal' : 'default' } %}
```

Comment:
The contents of `custom_styles.nav_type` come from here https://github.com/greenpeace/planet4-master-theme/blob/65fec738a6dc49f3f657a6d17cae89fde6f8ec48/single-campaign.php#L76-L76, and it's a string which can be either `minimal` or `'default`. Both would evaluate to true so it would use the `minimal` type if a page has `default` as a value.

Reply:
Got it! Thanks